pkgs/by-name/ly/lyssa/package.nix

```
@@ -0,0 +1,60 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  taglib,
+  cglm,
+  libclipboard,
+  libxcb,
+  makeWrapper,
+  python3Packages,
+  ffmpeg,
+  yt-dlp,
+  miniaudio,
+  jq,
+  glfw,
+  exiftool,
+  stb,
+  libGL
+}:
+
+stdenv.mkDerivation {
+  name = "lyssa";
+  version = "unstable-2024-06-01";
+
+  src = fetchFromGitHub {
+    owner = "cococry";
+    repo = "lyssa";
+    rev = "a7ebb786b005d9f716dfa99869e29660ca14c749";
+    hash = "sha256-jJx7V/ru6quvZhqkVK3OGguH81882fQXNdpLVp+g5sk=";
+  };
+
+  buildInputs = [ cglm libclipboard libxcb glfw ];
+  nativeBuildInputs = [ pkg-config taglib makeWrapper ];
+
+  postFixup = ''
+      wrapProgram $out/bin/lyssa \
+      --prefix PATH : ${
+      lib.makeBinPath [ jq exiftool stb miniaudio yt-dlp ffmpeg python3Packages.glad ]
+    }
+'';
```

maybe use a `let` block to avoid long `${...}` expression
```suggestion
  postFixup = let
    runtime-deps = lib.makeBinPath [
      jq
      exiftool
      stb
      miniaudio
      yt-dlp
      ffmpeg
      python3Packages.glad
    ];
  in ''
    wrapProgram $out/bin/lyssa \
      --prefix PATH : ${runtime-deps}
  '';
```